tests/data_adapter/test_geodataframe_adapter.py

```diff
@@ -78,7 +75,7 @@ def test_geodataframe_unit_attrs(self, artifact_data: DataCatalog):
         # gadm_level1_gdf = artifact_data.get_geodataframe("gadm_level1")
         # assert gadm_level1_gdf["NAME_0"].attrs["long_name"] == "Country names"
 
-    @pytest.mark.skip()  # FIXME
+    @pytest.mark.skip("Missing 'raster' driver implementation.")
```

Comment:
There are many integration tests in HydroMT, and not many unittests. I skip all integration tests that fail because some drivers are not implemented or other functionality is still waiting. I do implement some new unittests with every new feature.